pyslack/__init__.py

```diff
@@ -41,6 +41,22 @@ def chat_post_message(self, channel, text, **params):
         })
         return self._make_request(method, params)
 
+    def chat_update_message(self, channel, text, timestamp, **params):
+        """chat.update
+
+        This method updates a message.
```

Comment:
But they say "message to be updated," so if we're updating an updated message, it should be the timestamp of the updated message, right?